dbt/dbt-run.py

```diff
@@ -0,0 +1,81 @@
+import psycopg2
+import os
+import time
+import json
+import subprocess
+
+from urllib.parse import urlparse
+
+
+def connection():
+    for attempt in range(5):
+        time.sleep(3)
+        try:
+            return psycopg2.connect(
+                f"dbname={os.getenv('POSTGRES_DB')} "
+                f"user={os.getenv('POSTGRES_USER')} "
+                f"password={os.getenv('POSTGRES_PASSWORD')} "
+                f"host={os.getenv('POSTGRES_HOST') or 'postgres'} "
+                f"port={os.getenv('POSTGRES_PORT') or '5432'} "
+            )
+        except psycopg2.OperationalError as e:
+            print('Unable to connect!', e)
+        else:
+            break
+    raise Exception("Could not connect to postgres")
+
+
+# Create schema
+with connection() as conn:
+    with conn.cursor() as cur:
+        cur.execute(f"""
+            CREATE SCHEMA IF NOT EXISTS 
+            {os.getenv('POSTGRES_SCHEMA')}
+        """)
+    conn.commit()
+
+with connection() as conn:
+    with conn.cursor() as cur:
+        cur.execute(f"""
+            CREATE TABLE IF NOT EXISTS
+            {os.getenv('POSTGRES_SCHEMA')}._dataemon (
+                inserted_on TIMESTAMP DEFAULT NOW(),
+                packages jsonb
+            )
+        """)
+    conn.commit()
+
+init_package = urlparse(os.getenv("DATAEMON_INITAL_PACKAGE"))
```

Comment:
Let's add `DATAEMON_INITAL_PACKAGE` to the env.template file and provide a default value. Also there is a typo, it should be `DATAEMON_INITIAL_PACKAGE`

Reply:
yea, in the env file its called CHT_PIPELINE_BRANCH_URL and that ends up being used for DATAEMON_INITAL_PACKAGE in the docker compose yaml. so maybe we change the name everywhere to CHT_PIPELINE_BRANCH_URL